src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -228,8 +228,8 @@ export const DownloadCartPage:React.FunctionComponent<DownloadCartPageProps> = (
       <FullWidthAlert
         show={isShowingDownloadSuccessAlert}
         variant='success'
-        title='Package download' 
-        description='The files contained in this zip file have been removed from your list.'
+        title='Package Created' 
+        description='A package has been created with eligible files. The items contained in this .zip file have been removed from your list. If your package is over 1GB you will need to create multiple packages.'
```

Comment:
Super nitpicky, grammar
```suggestion
        description='A package has been created with eligible files. The items contained in this .zip file have been removed from your list. If your package is over 1GB, you will need to create multiple packages.'
```